.github/workflows/test-python-atlas-geo.yml

```diff
@@ -0,0 +1,59 @@
+# Identical to test-python-atlas.yml except that gdal-bin is also installed.
```

Comment:
Adding both  `.github/workflows/test-python-geo.yml` and `.github/workflows/test-python-atlas-geo.yml` might be redundant. (Are there any notable differences in GEO support on Atlas?) When the Atlas build ran with transactions, it was perhaps slightly more useful.

Reply:
I meant that `test-python-geo.yml` uses `supercharge/mongodb-github-action` (is this "standalone"?) and `test-python-atlas-geo.yml` uses the Atlas VM.